inventory/host_vars/www.openfoodfrance.org/config.yml

```diff
@@ -23,3 +23,6 @@ custom_hba_entries:
 
 enable_nginx_logging: true
 enable_rails_apm: "true" # has to be explicitly defined as a string due to some datadog bug
+
+# Images settings
+attachment_path: "home/openfoodnetwork/apps/openfoodnetwork/current/public/spree/products/:id/:style/:basename.:extension"
```

Comment:
This overriden path seems a bit odd. It includes the default `public/spree/products/:id/:style/:basename.:extension` path but prepends it with a relative path from the home folder. I'm not sure it would work. Can't we simply remove it so the default one applies?

Reply:
this is copied from the live DB. I thought you have done this :-) maybe @Matt-Yorkley ?

Reply:
we should give it a try

Reply:
Different production servers have different settings here, there's not a single value we can apply to all of them. It's about what setting was used when the server first started using S3, which wasn't always the default. All images for that server will now be stored with the given path. A couple of servers needed a different path prefix after the changes to symlinking in the deployment process...

Reply:
I am not sure why we need the full path here but I am going to keep it as is, it's not really the scope of this PR.